ios/RCCViewController.m

```diff
@@ -644,6 +644,16 @@ -(void)setStyleOnAppearForViewController:(UIViewController*)viewController appea
       self.navigationItem.titleView.clipsToBounds = YES;
     }
   }
+  
+  if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 11.0) {
```

Comment:
What do you think about using a `if (@available(iOS 11.0, *))` check here instead of checking the device's system version?

Reply:
Since the `@available` can only be used in xcode9 and higher (backward compatible wise), it's better to use device's system version

Reply:
If the argument is for backwards compatibility, I would argue that using `[self.navigationController.navigationBar respondsToSelector:@selector(setPrefersLargeTitles:)]` would be a better alternative than a version check.